### DISCUSSION TITLE
Version bump

### DISCUSSION THREAD

**Discussion by nikatsapien** [minio/minio-js#1348] [Q&A]
Hi team!

Love the product, and this package!

We are hoping to get a version with `fast-xml-parser` bumped to at least `4.4.1`, ideally under in version 7 (`7.x`). I see that dependency bumps have happened ([here](https://github.com/minio/minio-js/pull/1328) and [here](https://github.com/minio/minio-js/pull/1335)), but there have not been any `7.x` updates, let alone `8.x`, releases with this change.

Just checking in to see if we have a timeline for this.

**Comment by prakashsvmx** [comment]
We do not update packages for older versions. you can fork and update as per your preference.  we support for only latest versions.
